queries/nickel/highlights.scm

```diff
@@ -0,0 +1,64 @@
+(comment) @comment @spell
+
+[
+  "if"
+  "then"
+  "else"
```

Comment:
`@conditional` for if/else/then

Reply:
you already have the conditional highlight. I don't think keyword is necessary then

Reply:
Probably not, removed